_events/2017-04-09-this-bridge-called-my-back.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: this-bridge-called-my-back
```

Comment:
`identifier` is for the event itself while `reading_identifiers` maps to the readings. Most of our book clubs will have reading identifier that matches the event identifier, but in the future we may have 2+ readings for a single event.